stories/Select.tsx

```diff
@@ -139,8 +139,122 @@ storiesOf('Organisms|Select Components', module)
     // eslint-disable-next-line
     .add(
         'AutosuggestMulti',
-        ({ parameters }: any) => {
-            const store = parameters.getStore();
+        (storyContext) => {
+            const store = storyContext?.parameters.getStore();
+            const getSuggestions = (): TSuggestion[] => {
+                const suggestions = SUGGESTIONS.filter(
+                    (item: TSuggestion) =>
+                        item.name.toLocaleLowerCase() !==
+                            store.get('inputValue').toLocaleLowerCase() &&
```

Comment:
this condition should be removed because it doesn't allow to select an item once there's an exact match with the input value

Reply:
good catch